src/5.0_to_5.1/migrateCalculatedMeasuresToActivePivot/migrateCalculatedMeasuresInMdx.test.ts

```diff
@@ -0,0 +1,111 @@
+import { dataModelsForTests } from "@activeviam/data-model-5.0";
+import { MdxSelect, parse, stringify } from "@activeviam/mdx-5.0";
+import { migrateCalculatedMeasuresInMdx } from "./migrateCalculatedMeasuresInMdx";
+
+const mdxSelectWithNoCalculatedMeasures: MdxSelect = parse(
```

Comment:
It would be nice to use the indented version of the MDX so that it's easier to read.
Same for the test snapshots, we could use `stringify(mdx, {indent: true})`.
If it's too much of a hassle no worries though, it's not the biggest priority.